source/future.tex

```diff
@@ -110,57 +110,155 @@
+\tcode{<stdint.h>} \\
+\columnbreak
+\tcode{<stdio.h>} \\
+\tcode{<stdlib.h>} \\
+\tcode{<string.h>} \\
+\tcode{<tgmath.h>} \\
+\tcode{<time.h>} \\
+\tcode{<uchar.h>} \\
+\columnbreak
+\tcode{<wchar.h>} \\
+\tcode{<wctype.h>} \\
+\end{multicolfloattable}
+
+\rSec2[depr.complex.h.syn]{Header \tcode{<complex.h>} synopsis}
+
+\indexhdr{complex.h}%
+\begin{codeblock}
+#include <complex>
+\end{codeblock}
+
+\pnum
+The header \tcode{<complex.h>}
+behaves as if it simply includes the header
+\tcode{<complex>}\iref{complex.syn}.
+
+\pnum
+\begin{note}
+Names introduced by \tcode{<complex>} in namespace \tcode{std}
+are not placed into the global namespace scope by \tcode{<complex.h>}.
+\end{note}
+
+\rSec2[depr.iso646.h.syn]{Header \tcode{<iso646.h>} synopsis}
+
+\indexhdr{iso646.h}%
+\pnum
+The \Cpp{} header \tcode{<iso646.h>} is empty.
+\begin{note}
+\tcode{and},
+\tcode{and_eq},
+\tcode{bitand},
+\tcode{bitor},
+\tcode{compl},
+\tcode{not_eq},
+\tcode{not},
+\tcode{or},
+\tcode{or_eq},
+\tcode{xor}, and
+\tcode{xor_eq}
+are keywords in this International Standard\iref{lex.key}.
+\end{note}
+
+\rSec2[depr.stdalign.h.syn]{Header \tcode{<stdalign.h>} synopsis}
+
+\indexlibrary{\idxcode{__alignas_is_defined}}%
+\indexhdr{stdalign.h}%
+\begin{codeblock}
+#define @\xname{alignas_is_defined}@ 1
+\end{codeblock}
+
+\pnum
+\indexhdr{stdalign.h}%
+The contents of the \Cpp{} header \tcode{<stdalign.h>} are the same as the C
+standard library header \tcode{<stdalign.h>}, with the following changes:
+The header \tcode{<stdalign.h>} does not
+define a macro named \tcode{alignas}.
+
+\xrefc{7.15}
 
-\tcode{<assert.h>}			&
-\tcode{<inttypes.h>}		&
-\tcode{<signal.h>}			&
-\tcode{<stdio.h>}			  &
-\tcode{<wchar.h>}			  \\
+\rSec2[depr.stdbool.h.syn]{Header \tcode{<stdbool.h>} synopsis}
 
-\tcode{<complex.h>}			&
-\tcode{<iso646.h>}			&
-\tcode{<stdalign.h>}    &
-\tcode{<stdlib.h>}			&
-\tcode{<wctype.h>}			\\
+\indexhdr{stdbool.h}%
+\indexlibrary{\idxcode{__bool_true_false_are_defined}}%
+\begin{codeblock}
+#define @\xname{bool_true_false_are_defined}@ 1
+\end{codeblock}
+
+\pnum
+\indexhdr{stdbool.h}%
+The contents of the \Cpp{} header \tcode{<stdbool.h>} are the same as the C
+standard library header \tcode{<stdbool.h>}, with the following changes:
+The header \tcode{<stdbool.h>} does not
+define macros named \tcode{bool}, \tcode{true}, or \tcode{false}.
 
-\tcode{<ctype.h>}			  &
-\tcode{<limits.h>}			&
-\tcode{<stdarg.h>}			&
-\tcode{<string.h>}			& \\
+\xrefc{7.18}
 
-\tcode{<errno.h>}			  &
-\tcode{<locale.h>}			&
-\tcode{<stdbool.h>}			&
-\tcode{<tgmath.h>}			& \\
+\rSec2[depr.tgmath.h.syn]{Header \tcode{<tgmath.h>} synopsis}
 
-\tcode{<fenv.h>}			  &
-\tcode{<math.h>}			  &
-\tcode{<stddef.h>}			&
-\tcode{<time.h>}				& \\
+\indexhdr{tgmath.h}%
+\begin{codeblock}
+#include <cmath>
+#include <complex>
+\end{codeblock}
 
-\tcode{<float.h>}			  &
-\tcode{<setjmp.h>}			&
-\tcode{<stdint.h>}			&
-\tcode{<uchar.h>}			  & \\
+\pnum
+The header \tcode{<ctgmath>}
```

Comment:
Wait, this is weird -- why are we talking about `<ctgmath>` in the synopsis of `<tgmath.h>`?